genesis/process/shardGenesisBlockCreator.go

```diff
@@ -36,6 +37,11 @@ func CreateShardGenesisBlock(arg ArgsGenesisBlockCreator) (data.HeaderHandler, e
 	}
 
 	//TODO add here delegation process
+	if arg.HardForkConfig.MustImport {
+		//TODO think about how to integrate when genesis is modified as well for hardfork - when should set balances be called
```

Comment:
Is a must to keep the old, original genesis block? No one will care about it except the entities that want to sync from the start.

Reply:
agreed. at this moment it we do not care about initial genesis.